### PR TITLE
Reload own methods from source

### PR DESCRIPTION
**Pull request by lionel-**
Rather than from cache

### CODE REVIEW

File: R/load.r [hunk 1]
```diff
@@ -158,6 +158,12 @@ load_all <- function(path = ".", reset = TRUE, compile = NA,
       # methods. We'll restore the foreign methods but let the package
       # register its own methods again.
       old_methods <- as.list(methods_env)
+
+      is_foreign_method <- function(x) {
```
Comment (jimhester) [inline R/load.r:162]: Can you define this in the top level rather the in the body of the function?

Comment (lionel-) [reply R/load.r]: done; I thought this definition wasn't general enough to be at top level.

Comment (jimhester) [reply R/load.r]: That is fair, I just as a rule avoid defining functions within other function bodies, as it makes it hard to test them in isolation.

**Comment by jimhester** [comment]
Thanks!

**Pull request merged**